webrtc_patches/build_at_webrtc_branch_heads_52.patch

```diff
@@ -0,0 +1,51 @@
+From 79dda9f4fedd0df439f496b244a226167086cb61 Mon Sep 17 00:00:00 2001
+From: Cant Stop The Signal <cantstopthesignals@gmail.com>
+Date: Sun, 4 Dec 2016 16:01:43 -0800
+Subject: [PATCH] Add patches to get webrtc to build from webrtc
+ branch-heads/52
+
+webrtc commit f33698296719f956497d2dbff81b5080864a8804
+---
+ webrtc/api/api.gyp           |  3 +++
+ webrtc/api/fakefieldtrial.cc | 15 +++++++++++++++
+ 2 files changed, 18 insertions(+)
+ create mode 100644 webrtc/api/fakefieldtrial.cc
+
+diff --git a/webrtc/api/api.gyp b/webrtc/api/api.gyp
+index 377ef8f..01b16bc 100644
+--- a/webrtc/api/api.gyp
++++ b/webrtc/api/api.gyp
+@@ -199,6 +199,9 @@
+         'webrtcsession.h',
+         'webrtcsessiondescriptionfactory.cc',
+         'webrtcsessiondescriptionfactory.h',
++        'test/fakeaudiocapturemodule.cc',
++        'test/fakeaudiocapturemodule.h',
++        'fakefieldtrial.cc',
```

Comment:
Let's just add `'<(webrtc_root)/system_wrappers/system_wrappers.gyp:field_trial_default',` to the `'dependencies'` like in `libjingle_peerconnection_jni` instead of providing our own.